package.json

```diff
@@ -40,7 +40,7 @@
     "simput": "1.2.0",
     "pvw-visualizer": "1.0.11",
 
-    "kw-web-suite": "1.1.0"
+    "kw-web-suite": "git://github.com/Kitware/kw-web-suite.git#reduced-hpcc-deps"
```

Comment:
I think you can write it like that:

Kitware/kw-web-suite.git#reduced-hpcc-deps